.github/workflows/callable-qa.yml

```diff
@@ -174,7 +174,7 @@ jobs:
                 name: Contribution is under MIT and has no merge commits
                 if: "always() && steps.checkout.outcome == 'success'"
                 run: |
-                    .github/recipes-checker-main/run lint:pull-request $GITHUB_EVENT_PATH ${{ secrets.GITHUB_TOKEN }}
+                    .github/recipes-checker-main/run lint:pull-request --license MIT $GITHUB_EVENT_PATH ${{ secrets.GITHUB_TOKEN }}
```

Comment:
in your PR on the checker, `MIT` is the default value of the option, so this change is currently useless.

Reply:
I left MIT there as default value to keep BC just in case.
Here I decided to specify it explicitly